src/main/java/io/fusionauth/domain/event/EmailTemplateTestEvent.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2021, FusionAuth, All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+package io.fusionauth.domain.event;
+
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.Collections;
+import java.util.Objects;
+
+import com.inversoft.json.JacksonConstructor;
+import com.inversoft.json.ToString;
+import io.fusionauth.domain.Buildable;
+import io.fusionauth.domain.EventInfo;
+import io.fusionauth.domain.Location;
+
+/**
+ * @author Brett Guy
+ */
+public class EmailTemplateTestEvent extends BaseEvent implements Buildable<EmailTemplateTestEvent> {
+  public String message;
+
+  @JacksonConstructor
+  public EmailTemplateTestEvent() {
```

Comment:
I think we can pass in the `EventInfo` so the caller can provide the actual values.